app/models/katello/glue/candlepin/pool.rb

```diff
@@ -93,24 +123,13 @@ def provider?(organization)
 
       # rubocop:disable Metrics/MethodLength,Metrics/AbcSize
       # rubocop:disable Metrics/CyclomaticComplexity
-      # rubocop:disable Metrics/PerceivedComplexity
       def import_data(index_hosts_and_activation_keys = false)
```

Comment:
This method was written in a very confusing way, especially with the lack of validations around org and subscription ids, and non-nil constraints. It gives the impression that a Pool can change organizations or Subscriptions and I don't think that's ever valid.